### PR TITLE
Playwright: Update dependencies, support HA server and fix tests

### PR DESCRIPTION
**Pull request by saturninoabril**
#### Summary
- Update dependencies, support HA server and fix tests.
- Introduce `matchSnapshot` that consolidates matching snapshot thru built-in snapshot of Playwright, PercyIO and Applitools depending on whichever is enabled.

#### Ticket Link
none

#### Release Note
```release-note
NONE
```

### CODE REVIEW

**Review by josephbaylon — APPROVED** [review]
LGTM. Thanks @saturninoabril !

File: e2e/playwright/global_setup.ts [hunk 1]
```diff
@@ -123,6 +118,35 @@ async function sysadminSetup(client: Client, user: UserProfile) {
...
+
+        const {Enable, ClusterName} = (await client.getConfig()).ClusterSettings;
+        expect(Enable, Enable ? '' : 'Should have cluster enabled').toBe(true);
+
+        const sameClusterName = ClusterName === haClusterName;
+        expect(
+            sameClusterName,
+            sameClusterName
+                ? ''
+                : `Should have cluster name set and as expected. Got "${ClusterName}" but expected "${ClusterName}"`
```
Comment (Phrynobatrachus) [inline e2e/playwright/global_setup.ts:134]:
```suggestion
                : `Should have cluster name set and as expected. Got "${ClusterName}" but expected "${haClusterName}"`
```
I think?

**Review by Phrynobatrachus — APPROVED** [review]
Cool! `matchSnapshot` is a nice refactor.

File: e2e/playwright/global_setup.ts (hunk 1 above)
Comment (saturninoabril) [reply e2e/playwright/global_setup.ts]: Good catch, thanks Cass!

**Review by nevyangelova — APPROVED** [review]
TYSM for handling this!

**Comment by saturninoabril** [comment]
@mvitale1989 This should be good to merge now since there's enough approvers already but let me know if you're looking into this before merging. TIA.

**Comment by mvitale1989** [comment]
Sounds good, don't want to block this any further. Sorry for the delay!

**Pull request merged**